.github/labeler.yml

```diff
@@ -82,12 +57,8 @@ SPARK SHELL:
   - "repl/**/*"
   - "bin/spark-shell*"
 SQL:
-#- any: ["**/sql/**/*", "!python/pyspark/sql/avro/**/*", "!python/pyspark/sql/streaming/**/*", "!python/pyspark/sql/tests/streaming/test_streaming.py"]
-  - "**/sql/**/*"
+  - any: ["**/sql/**/*", "!python/pyspark/sql/avro/**/*", "!python/pyspark/sql/streaming/**/*", "!python/pyspark/sql/tests/streaming/test_streaming*.py"]
```

Comment:
Multiple files existed in the path:
<img width="1085" alt="image" src="https://github.com/apache/spark/assets/15246973/48a97370-e6e5-4f8a-9c29-af0cfb046511">
So `test_streaming.py` -> `test_streaming*.py`

https://github.com/apache/spark/tree/d582b74d97a7e44bdd2f0ae6c63121fc5e5466b7/python/pyspark/sql/tests/streaming